### PR TITLE
Update Automation Registry/Registrar addresses + Migrate to Arbitrum Sepolia

### PR DESCRIPTION
**Pull request by ritika-b**
- updates Automation registry and registrar addresses
- updates Goerli to Sepolia
- updates Getting Started

### CODE REVIEW

File: src/features/chainlink-automation/data/chainlink-automation-config.json [hunk 1]
```diff
@@ -321,7 +321,7 @@
     "maxPerformDataSize": 2000,
     "flatFeeMicroLink": 0,
     "stalenessSeconds": 90000,
-    "registrar": "0x1433C1BDfCF1b2E1CE843Ede6FcC5d6e409fE56E",
+    "registrar": "0xaB9718E1DC7BD61Bd3440Fb771CC23DA0C0181a1",
```
Comment (thedriftofwords) [inline src/features/chainlink-automation/data/chainlink-automation-config.json:324]: Should this be the registrar address `0xA6deE996014d9670144943c2bc90755829398d07` as shown in your change to [chainlink-automation-addresses.json](https://github.com/smartcontractkit/documentation/pull/1647/files#diff-c5baa47f887aec758a3420ae29244422e1f111a0178aa9fc8b0ba2381ffcf5ff)?

**Review by thedriftofwords — CHANGES_REQUESTED** [review]

File: src/features/chainlink-automation/data/chainlink-automation-config.json (hunk 1 above)
Comment (thedriftofwords) [inline src/features/chainlink-automation/data/chainlink-automation-config.json:324]:
```suggestion
    "registrar": "0xA6deE996014d9670144943c2bc90755829398d07",
```

**Review by dwightjl — APPROVED** [review]

File: src/content/chainlink-automation/overview/getting-started.mdx [hunk 2]
```diff
@@ -33,13 +33,13 @@ Click the tabs below to use Chainlink Automation with each type of trigger:
   <Fragment slot="panel.timeBased">
       **Increment a counter every 5 minutes using our example contract.**
 
-      1. Navigate to the [Chainlink Automation app](https://automation.chain.link/) and connect to **Arbitrum Goerli** in the top dropdown menu.
+      1. Navigate to the [Chainlink Automation app](https://automation.chain.link/) and connect to **Arbitrum Sepolia** in the top dropdown menu.
 
-      2. Connect your cryptocurrency wallet to the app if you haven't done so already. You may also need to fetch Arbitrum Goerli testnet LINK [here](https://faucets.chain.link/arbitrum-goerli).
+      2. Connect your cryptocurrency wallet to the app if you haven't done so already. You may also need to fetch Arbitrum Sepolia testnet LINK [here](https://faucets.chain.link/).
```
Comment (dwightjl) [inline src/content/chainlink-automation/overview/getting-started.mdx:38]:
```suggestion
      2. Connect your cryptocurrency wallet to the app if you haven't done so already. You may also need to fetch Arbitrum Sepolia testnet LINK [here](https://automation.chain.link/arbitrum-sepolia).
```

File: src/content/chainlink-automation/overview/getting-started.mdx [hunk 3]
```diff
@@ -63,13 +63,13 @@ Click the tabs below to use Chainlink Automation with each type of trigger:
   <Fragment slot="panel.customLogic">
       **Increment a counter using custom logic stored on-chain.**
 
-      1. Navigate to the [Chainlink Automation app](https://automation.chain.link/) and connect to **Arbitrum Goerli** in the top dropdown menu.
+      1. Navigate to the [Chainlink Automation app](https://automation.chain.link/) and connect to **Arbitrum Sepolia** in the top dropdown menu.
 
-      2. Connect your cryptocurrency wallet to the app if you haven't done so already. You may also need to fetch Arbitrum Goerli testnet LINK [here](https://faucets.chain.link/arbitrum-goerli).
+      2. Connect your cryptocurrency wallet to the app if you haven't done so already. You may also need to fetch Arbitrum Sepolia testnet LINK [here](https://faucets.chain.link/).
```
Comment (dwightjl) [inline src/content/chainlink-automation/overview/getting-started.mdx:68]:
```suggestion
      2. Connect your cryptocurrency wallet to the app if you haven't done so already. You may also need to fetch Arbitrum Sepolia testnet LINK [here](https://automation.chain.link/arbitrum-sepolia).
```

File: src/content/chainlink-automation/overview/getting-started.mdx [hunk 4]
```diff
@@ -90,15 +90,15 @@ Click the tabs below to use Chainlink Automation with each type of trigger:
 
       **Increment an on-chain counter using a log as trigger.**
 
-      1. Navigate to the [Chainlink Automation app](https://automation.chain.link/) and connect to **Arbitrum Goerli** in the top dropdown menu.
+      1. Navigate to the [Chainlink Automation app](https://automation.chain.link/) and connect to **Arbitrum Sepolia** in the top dropdown menu.
 
-      2. Connect your cryptocurrency wallet to the app if you haven't done so already. You may also need to fetch Arbitrum Goerli testnet LINK [here](https://faucets.chain.link/arbitrum-goerli).
+      2. Connect your cryptocurrency wallet to the app if you haven't done so already. You may also need to fetch Arbitrum Sepolia testnet LINK [here](https://faucets.chain.link/).
```
Comment (dwightjl) [inline src/content/chainlink-automation/overview/getting-started.mdx:95]:
```suggestion
      2. Connect your cryptocurrency wallet to the app if you haven't done so already. You may also need to fetch Arbitrum Sepolia testnet LINK [here](https://automation.chain.link/arbitrum-sepolia).
```

**Pull request merged**